### ISSUE TITLE
Typescript Support

### ISSUE THREAD

**Issue by d0mmi** [JonnyC0G/roccat-led#3]
Would it be possible for you to add TypeScript type definitions?

**Comment by JonnyC0G** [comment]
I don't have much experience with typescript, but I'll consider it :)

**Comment by JonnyC0G** [comment]
@d0mmi the new published version  `0.2.6` contains typescript definitions, although I still don't know what exactly is happening there. 
As far as I tested there should no breaking changes in functionality, but unfortunately the files are now wrapped in the dist-folder.

**Comment by d0mmi** [comment]
> @d0mmi the new published version  `0.2.6` contains typescript definitions, although I still don't know what exactly is happening there. 
> As far as I tested there should no breaking changes in functionality, but unfortunately the files are now wrapped in the dist-folder.

![Screenshot_20211228-082440.jpg](https://user-images.githubusercontent.com/16517043/147539902-037eaf4a-f013-4eeb-8b4e-78fcd004ebfe.jpg)
That's why they are in the dist folder

**Comment by JonnyC0G** [comment]
yeah i know that. the module is coded in ts, but i wanted to publish just the compiled js files. and for obvious reasons i don't want my compiles sources in the project root.
